frontend/src/pages/projects/screens/detail/storage/ClusterStorageTableRow.tsx

```diff
@@ -0,0 +1,268 @@
+                ? `${MOUNT_PATH_PREFIX}${clusterStorageName.toLowerCase().replace(/\s+/g, '-')}-${
+                    rowIndex + 1
+                  }`
+                : '',
+              error: '',
+            }
+          : undefined,
+      );
+    }
+  }, [
+    addNotebookData,
+    clusterStorageName,
+    existingPVC,
+    obj.existingValue,
+    obj.mountPath.value,
+    rowIndex,
+  ]);
+
+  const suffix = mountPathSuffix(obj.mountPath.value);
+  const format = mountPathFormat(obj.mountPath.value);
+
+  const allInUseMountPaths = getNotebookPVCMountPathMap(
+    notebooks.find((notebook) => notebook.metadata.name === obj.name),
+  );
+
+  const inUseMountPaths = Object.keys(allInUseMountPaths)
+    .filter((key) => key !== existingPVC)
+    .map((key) => allInUseMountPaths[key]);
+
+  const pathFormatOptions = [
+    {
+      type: MountPathFormat.STANDARD,
+      description: `Standard paths that begins with ${MOUNT_PATH_PREFIX} are visible in JupyterLab file browser.`,
+    },
+    {
+      type: MountPathFormat.CUSTOM,
+      description: `Custom paths that do not begin with ${MOUNT_PATH_PREFIX} are not visible in the JupyterLab file browser.`,
+    },
+  ];
+
+  // Define options for notebook selection
+  const selectOptions = React.useMemo(() => {
+    const options = availableNotebooks.map((notebook) => ({
+      value: notebook.metadata.name,
+      content: notebook.metadata.name,
+    }));
+
+    if (obj.name) {
+      options.push({
+        value: obj.name,
+        content: obj.name,
+      });
+    }
+
+    return options;
+  }, [availableNotebooks, obj.name]);
+
+  const validateAndUpdate = React.useCallback(
+    (value: string, newFormat?: 'Standard' | 'Custom') => {
+      const prefix = (newFormat ?? format) === MountPathFormat.STANDARD ? MOUNT_PATH_PREFIX : '';
+      const newValue = `${prefix}${value}`;
+      addNotebookData(
+        obj.name,
+        {
+          value: newValue,
+          error: validateClusterMountPath(newValue, inUseMountPaths) || '',
+        },
+        true,
+      );
+    },
+    [addNotebookData, format, inUseMountPaths, obj.name],
+  );
+
+  const handleFormatChange = (newFormat: 'Standard' | 'Custom') => {
+    validateAndUpdate(suffix, newFormat);
+  };
+
+  return (
+    <Tbody>
+      <Tr>
+        <Td dataLabel="Name" style={{ paddingLeft: '5px' }}>
+          <TypeaheadSelect
+            shouldFocusToggleOnSelect
+            placeholder="Select a workbench"
+            selectOptions={selectOptions}
+            selected={obj.name}
+            onClearSelection={() => addNotebookData('')}
+            onSelect={(_ev, selectedValue) => {
+              if (typeof selectedValue === 'string') {
+                addNotebookData(selectedValue);
+              }
+            }}
+          />
+        </Td>
+        <Td dataLabel="Path format">
+          <SimpleSelect
+            isFullWidth
+            popperProps={{ direction: 'down' }}
```

Comment:
You can also add `maxWidth: undefined` here to override the default `maxWidth: 'trigger'` in the SimpleSelect so you can see the options menu show the correct width. Otherwise, it will be limited the width same as the toggle width.

Reply:
@DaoDaoNoCode thanks, I have updated it to `maxWidth: undefined`  but the width is limited to some extend i have a added `width:30vw`